docs/source/generate_commands.py

```diff
@@ -110,7 +110,7 @@ def get_mode_docs():
             modefile.write('The following commands are available in %s mode'
                            '\n\n' % mode)
         else:
-            header = 'Global Commands'
+            header = 'Global commands'
```

Comment:
This is a title, please leave `commands` capitalized.

Reply:
None of the sibling titles are capitalized (and I see no way to change this without making the whole thing ugly):

- 2.4 Commands
    - 2.4.1 Global Commands
    - 2.4.2 Commands in _bufferlist_ mode
    - 2.4.3 Commands in _envelope_ mode
    - 2.4.4 Commands in _namedqueries_ mode
    - _&c._

Perhaps I should fix the (non-)capitalization of 2.1, though.

Reply:
Okay, it's annoying that the TOC renders it that way, because it does look somewhat unnatural there.

Reply:
I actually feel all right without titlecase for tertiary headings. My native language doesn’t use titlecase at all, though.

What do you think should be done with the tertiary titles in section 3? Currently about a half of them are titlecased.